xml/art-obs-beginners-guide.xml

```diff
@@ -204,46 +200,45 @@ https://www.suse.com/communities/blog/suse-studio-integration/
  </sect1>
 
  <sect1 xml:id="sec.obsbg.req">
-  <title>Requirements</title>
+  <title>Requirements for Working with &osccmd; on the Command Line</title>
```

Comment:
I know but it is more correct nonetheless. Especially, since there is also a Prerequisites section and those two looked kinda similar from their titles.